source/AI.cpp

```diff
@@ -4071,6 +4073,7 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 			if(!target)
 			{
 				message = "There are no planets in this system that you can land on.";
+				messageImportance = Messages::Importance::Highest;
 				Audio::Play(Audio::Get("fail"));
 			}
 			else if(!target->GetPlanet()->CanLand())
```

Comment:
I meant this message when I said "the authorities refuse to let you land."

Reply:
I'm sorry, I don't get this. 
Is it fine now?
Importance changed for "There are no planets in this system that you can land on." and
refuse to let you land."

Reply:
@Amazinite Got it. Done.

Reply:
You've changed the importance for the authorities message in Engine, but there are two other locations here in AI that can send the same message under different circumstances that still have the High importance instead of the Highest importance.

Reply:
Your latest commit got one of them, but there's another one on line 4036.

Reply:
Found second entry. Changed.